### DISCUSSION TITLE
Need help: How can I add Hashnode blog's banner image?

### DISCUSSION THREAD

**Discussion by priyankarpal** [gautamkrishnar/blog-post-workflow#196] [Q&A]
I tried but I can't add the hashnode blog's banner image. How can I add them? I just added blogs.
Also, I want to create a table such as

Banner image | Title  | Description |
| -------- | ------ | ----------------- | 
| blog 1 banner | blog 1  | blog 1 desc |
| blog 2 banner | blog 2   | blog 2 desc |

![Screenshot from 2023-02-18 01-15-27](https://user-images.githubusercontent.com/88102392/219777149-78773250-cb06-4f00-a4e4-ab09a4678331.png)

**Comment by gautamkrishnar** [comment]
Looks like you had already achieved it using another action. People can easily achieve this by using the same code that is mentioned in the Youtube videos table shown in the readme. Some tweaking is necessary for hash node images to show up.  https://github.com/gautamkrishnar/blog-post-workflow#advanced-usage-examples

**Reply by Sagar0-0** [reply]
Is it possible to add medium post banner image?

**Reply by Sagar0-0** [reply]
The youtube example works because it gives us the flexibility to fetch the thumbnail with that specific URL, but for other platforms, how this will be possible?
